### DISCUSSION TITLE
Add new backend languages

### DISCUSSION THREAD

**Discussion by PabloKench** [tauri-apps/tauri#11721] [Ideas]
Well, I was currently looking at the possibility of changing my maui app and another electron to tauri

I noticed that the back end is in rust, I'm not very familiar with the language. Are you thinking about adding other ways to write the back end?

Do we now have good languages ​​like **C#, C++** and **Python**?

I noticed that it has Blazor webassembly, but the use of webassembly is limited to using **.NET** functions
I really liked tauri, but I don't like the rust language, is there any possibility of supporting other languages ​​for the backend in the future?

**Comment by FabianLars** [comment]
We would love to add new languages in the future, yes. We currently just don't have the time and/or people to work on this. Here's our tracking issue https://github.com/tauri-apps/tauri/issues/4670

So for now the only alternative is having a separate app in the language of your choice and bundling it as a "sidecar" alongside your app :/
